docs/UserGuide.md

```diff
@@ -90,6 +90,23 @@ Shows a list of all persons in the address book.
 
 Format: `list`
 
+### Filter students by tags: `filter`
+
+Filters students who have a given tag.
+
+Format: `filter t/TAG`
```

Comment:
I think this would be a bit confusing because the students have a TAG field. I imagine you mean that the t/TAG is replaced by something else like n/STUDENT_NAME right? 

Maybe you could do filter [n/STUDENT_NAME] [ev/EVENT_NAME] where [ ] means optional field. You could then add a comment saying at least one field is required.

Maybe we could use Command Format instead of Format as that would be clearer?